test/test_files/nalu-nalu-panel/nalu_platform.yaml

```diff
@@ -0,0 +1,246 @@
+            max_iterations: 1
+            convergence_tolerance: 1e-7
+
+        - ShearStressTransport:
+            name: mySST 
+            max_iterations: 1
+            convergence_tolerance: 1e-8
+
+    initial_conditions:
+
+      - constant: ic_1
+        target_name: [bulk-HEX]
+        value:
+          pressure: 0.0
+          velocity: [51.8,0.0,0.0]
+          turbulent_ke: 0.095118
+          specific_dissipation_rate: 250.0
+
+    material_properties:
+      target_name: [bulk-HEX]
+      specifications:
+        - name: density
+          type: constant
+          value: 1.20
+
+        - name: viscosity
+          type: constant
+          value: 1.8e-5
+
+    boundary_conditions:
+
+    - inflow_boundary_condition: bc_inflow
+      target_name: front
+      inflow_user_data:
+        velocity: [58.1, 0.0, 0.0]
+        turbulent_ke: 0.095118
+        specific_dissipation_rate: 250.0
+
+    - symmetry_boundary_condition: bc_pz
+      target_name: top
+      symmetry_user_data:
+
+    - symmetry_boundary_condition: bc_py
+      target_name: plus_y
+      symmetry_user_data:
+
+    - symmetry_boundary_condition: bc_my
+      target_name: minus_y
+      symmetry_user_data:
+
+    - wall_boundary_condition: bc_bottom
+      target_name: ground
+      wall_user_data:
+        velocity: [0.0, 0.0, 0.0]
+        turbulent_ke: 0.0
+
+    - wall_boundary_condition: bc_bottom
+      target_name: structure
+      wall_user_data:
+        velocity: [0.0, 0.0, 0.0]
+        turbulent_ke: 0.0
+
+    - open_boundary_condition: bc_out
+      target_name: back
+      open_user_data:
+        velocity: [0,0]
+        pressure: 0.0
+        turbulent_ke: 0.095118
+        specific_dissipation_rate: 250.0
+    
+    - overset_boundary_condition: bc_ovst
+      overset_connectivity_type: tioga
+      overset_user_data:
+        mesh_tag_offset: 0
+        tioga_options:
+          set_resolutions: no
+
+        mesh_group:
+          - overset_name: background
+            mesh_parts: [ bulk-HEX ]
+            wall_parts: [ ground, structure ]
+
+    solution_options:
+      name: myOptions
+      turbulence_model: sst
+      projected_timescale_type: momentum_diag_inv #### Use 1/diagA formulation
+
+      options:
+        - hybrid_factor:
+            velocity: 1.0
+            turbulent_ke: 1.0
+            specific_dissipation_rate: 1.0
+
+        - upw_factor:
+            velocity: 0.0
+            turbulent_ke: 0.0
+            specific_dissipation_rate: 0.0
+
+        - alpha_upw:
+            velocity: 1.0
+            turbulent_ke: 1.0
+            specific_dissipation_rate: 1.0
+
+        - limiter:
+            pressure: no
+            velocity: no
+
+        - projected_nodal_gradient:
+            pressure: element
+            velocity: element
+
+        - relaxation_factor:
+            velocity: 0.7
+            pressure: 0.3
+            turbulent_ke: 0.7
+            specific_dissipation_rate: 0.7
+    # post_processing:
```

Comment:
Let's delete all these commented things if we don't need them

Reply:
Deleted the commented out options.